picasso/src/test/java/com/squareup/picasso/BitmapHunterTest.java

```diff
@@ -389,7 +389,8 @@
         dispatcher, cache, stats, action1);
     hunter.attach(action2);
     assertThat(hunter.getAction()).isEqualTo(action1);
-    assertThat(hunter.getActions()).hasSize(1).contains(action2);
+    assertThat(hunter.getActions()).hasSize(1);
+    assertThat(hunter.getActions()).contains(action2);
```

Comment:
Could probably replace these hasSize+contains calls with containsExactly, but nbd, this is what the existing code had.

Reply:
Yeah, I agree it's cleaner to read this way. I actually tried to preserve everything as close as it was.
Made a new commit that replaces instances of this case.